src/dotnet/CarbonAware.WebApi.Tests/unitTests/TestHelpers.cs

```diff
@@ -0,0 +1,27 @@
+namespace CarbonAware.WepApi.UnitTests
+{
```

Comment:
nit: I think you don't need anymore the `{}` for `namespace` using .net 6.0. Take a look at the other .cs files.